app/services/dogs_service.rb

```diff
@@ -0,0 +1,28 @@
+class DogsService
+  def self.conn
+    Faraday.new(url: "http://localhost:3000/")
+  end
+  
+  def self.get_url(url)
+    response = conn.get(url)
+    JSON.parse(response.body, symbolize_names: true)
+  end  
```

Comment:
Nice Job with the way you built out the service and broke out the methods when reaching out to the endpoints.